sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -84,6 +84,9 @@ class BlobClient(StorageAccountHostsMixin):  # pylint: disable=too-many-public-m
         account URL already has a SAS token. The value can be a SAS token string, an account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
         If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword str api_version:
```

Comment:
can we make it: default to the most recent one?

Reply:
It already does default to the latest one and is documented as such:
https://github.com/Azure/azure-sdk-for-python/pull/9635/files/ac828198c0d3fa06d9c8cdb8803ed7d99abc6197#diff-cfe8db5b056e62c6775c15a776384beaR158

Reply:
Ah sorry - I misunderstood you comment, you mean update the docstring to say "Default API version will be the latest"
Will this always be the case though? Maybe there's a new API version and we don't have a new SDK for it straight away?

Reply:
ahh probably sometimes... While ideally for every service release there's supposed to be an SDK release. Let's just leave it there then

---
In reply to: [374977713](https://github.com/Azure/azure-sdk-for-python/pull/9635#discussion_r374977713) [](ancestors = 374977713)